cookbooks/aws-parallelcluster-slurm/resources/munge/partial/_munge_actions.rb

```diff
@@ -0,0 +1,118 @@
+# frozen_string_literal: true
+
+#
+# Cookbook:: aws-parallelcluster-slurm
+# Recipe:: install_munge
+#
+# Copyright:: 2013-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Munge
+munge_version = '0.5.14'
+munge_url = "https://github.com/dun/munge/archive/munge-#{munge_version}.tar.gz"
+munge_tarball = "#{node['cluster']['sources_dir']}/munge-#{munge_version}.tar.gz"
+munge_user = 'munge'
+munge_user_id = node['cluster']['reserved_base_uid'] + 2
```

Comment:
What about defining a `reserved_uid_offset_munge` parameter equal to 2 in the global `default.rb` and then referencing it here?

Reply:
Would make sense,
however after working at the install_slurm PR I discovered that we have also other user/groups related to Slurm.
So I'd prefer to define a `base_slurm_user_id` (and possibly a reserved interval of ids) and then define the slurm user, munge user, and so on at given offsets.

I didn't do it yet because I still don't have a full picture of how many users are required.

Reply:
You can gain a full picture by looking at the number of UUID variables in the big old `default.rb`. For Slurm we should need 2 users, and around 10 UUIDs or less in total.